.nuget/create_nuget.py

```diff
@@ -66,11 +66,11 @@
     target_str = '''
     <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
     <PropertyGroup>
-        <EnableMLUnsupportedPlatformTargetCheck Condition="'$(EnableMLUnsupportedPlatformTargetCheck)' == ''">true</EnableMLUnsupportedPlatformTargetCheck>
+        <EnableLightGBMUnsupportedPlatformTargetCheck Condition="'$(EnableLightGBMUnsupportedPlatformTargetCheck)' == ''">true</EnableLightGBMUnsupportedPlatformTargetCheck>
     </PropertyGroup>
-    <Target Name="_CheckForUnsupportedPlatformTarget"
-            Condition="'$(EnableMLUnsupportedPlatformTargetCheck)' == 'true'"
-            AfterTargets="_CheckForInvalidConfigurationAndPlatform">
+    <Target Name="_LightGBMCheckForUnsupportedPlatformTarget"
+            Condition="'$(EnableLightGBMUnsupportedPlatformTargetCheck)' == 'true'"
+            AfterTargets="_LightGBMCheckForInvalidConfigurationAndPlatform">
```

Comment:
This one actually needs to stay the same:  `AfterTargets="_CheckForInvalidConfigurationAndPlatform"`.

`AfterTargets` is an MSBuild extensiblity mechanism to say "run this target after you run this other target".  See https://docs.microsoft.com/en-us/visualstudio/msbuild/target-element-msbuild#attributes-and-elements and https://msdn.microsoft.com/en-us/library/ee216359.aspx.

 `_CheckForInvalidConfigurationAndPlatform` is a well-known target that is defined in the common MSBuild files.